app-crypt/mc-agent/files/mc-agent.init

```diff
@@ -0,0 +1,18 @@
+#!/sbin/openrc-run
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v3
+
+SSH_AUTH_SOCK="$SOCK_DIR/$SOCK_FILE"
+
+command="/usr/bin/mc-agent"
+command_args="--address=${SSH_AUTH_SOCK} --no-fork"
+pidfile=/var/run/mc-agent.pid
```

Comment:
just use /run.
and if you will also move socket file to /run, it will need a sub-directory here in /run to hold pid and soket file.
so it'll become
`pidfile=/var/run/mc-agent/mc-agent.pid`